hr_holidays_imposed_days/models/hr_imposed_holidays.py

```diff
@@ -59,7 +59,11 @@ def validate(self):
                 leave._onchange_date_from()
                 created |= leave
             if rec.auto_confirm:
-                created.action_validate()
+                need_double_validation = created.filtered(
+                    lambda h: h.double_validation)
+                created.action_approve()
+                if need_double_validation:
+                    created.action_validate()
```

Comment:
Shouldn't this line be `need_double_validation.action_validate()`?

Reply:
Yes, you are absolutely right! Thank you for your review! :)